internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,89 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
 	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	Personal personaldata.Personal
+}
+
+func (ds DaySteps) Weight() float64 {
+	return ds.Personal.Weight
 }
 
-func (ds *DaySteps) Parse(datastring string) (err error) {
+func (ds DaySteps) Height() float64 {
+	return ds.Personal.Height
+}
+
+func (ds *DaySteps) Parse(datastring string) error {
 	// TODO: реализовать функцию
+	parts := strings.Split(datastring, ",")
+	if len(parts) != 2 {
+		return errors.New("Неверный формат данных, ожидается 'шаги, длительность'")
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return fmt.Errorf("Ошибка парсинга шагов: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("Ошибка парсинга шагов: %w", err)
```